bin/zowe-init.sh

```diff
@@ -135,13 +135,8 @@ else
     echo "  ZOWE_ZOSMF_PORT variable value="$ZOWE_ZOSMF_PORT >> $LOG_FILE
 fi
 
-if [[ -z ${JAVA_HOME} ]]
-then
-    JAVA_HOME=/usr/lpp/java/J8.0_64
-else
-    echo "  JAVA_HOME variable value="${JAVA_HOME} >> $LOG_FILE
-fi
-locateJavaHome ${JAVA_HOME}
```

Comment:
Do we still need `locateJavaHome` and `getJavaVersion` function in this file?

Reply:
Thanks Jack - I missed that from the clean up!